WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsModalViewController.swift

```diff
@@ -293,7 +295,9 @@ private extension CardPresentPaymentsModalViewController {
             auxiliaryButton.setAttributedTitle(viewModel.auxiliaryAttributedButtonTitle, for: .normal)
             auxiliaryButton.setImage(viewModel.auxiliaryButtonimage, for: .normal)
             if viewModel.auxiliaryButtonimage != nil {
-                auxiliaryButton.distributeTitleAndImage(spacing: 8.0)
+                var config = UIButton.Configuration.plain()
+                config.imagePadding = CGFloat(Constants.buttonTitleAndImageSpacing)
```

Comment:
nit: `buttonTitleAndImageSpacing` is already declared as `CGFloat`, no need to wrap/convert.
Applies also to 3 similar cases below.

Reply:
> nit: `buttonTitleAndImageSpacing` is already declared as `CGFloat`, no need to wrap/convert. Applies also to 3 similar cases below.

Oops! Removed on 3d9760bd7208e955a12daa8b56673924a97b530c